<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Module_MOD1103_Condition</class>
 <widget class="QMainWindow" name="Module_MOD1103_Condition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MFS1103-Thực hành</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../image.qrc">
    <normaloff>:/image/MFS0103_icon.jpg</normaloff>:/image/MFS0103_icon.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(44, 58, 70);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget_CAR01">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1031</width>
      <height>521</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget::pane { 
border: 1px solid #C0C0C0;
}
</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>       Thực hành       </string>
     </attribute>
     <widget class="QLabel" name="label_Blink_Stop">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>10</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(85, 170, 255);</string>
      </property>
      <property name="text">
       <string>Điều kiện kết thúc</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Blink_Process">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>371</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Quá trình</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_runBlink">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>58</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>410</y>
        <width>900</width>
        <height>1</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QFrame" name="frame_Condition">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>1011</width>
        <height>231</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_3">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>10</y>
         <width>381</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
border:1px solid gray;
background-color: rgb(44, 58, 70);
color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLineEdit" name="lineEdit_Condition_Stop">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>50</y>
          <width>91</width>
          <height>32</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="maxLength">
         <number>4</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_Condition_Stop_Car">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>10</y>
          <width>91</width>
          <height>32</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox {
    border: 1px solid gray;
	color: rgb(255, 255, 255);
}
QComboBox::drop-down {
    width:35px;
}</string>
        </property>
        <item>
         <property name="text">
          <string>XE-01</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>XE-02</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TẤT CẢ</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>14</y>
          <width>191</width>
          <height>22</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(85, 170, 255);</string>
        </property>
        <property name="text">
         <string>Điều kiện kết thúc cho</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_Condition_Stop">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>191</width>
          <height>32</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox {
    border: 1px solid gray;
	color: rgb(255, 255, 255);
}
QComboBox::drop-down {
    width:35px;
}</string>
        </property>
        <item>
         <property name="text">
          <string>Không có điều kiện</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Quãng đường (lớn hơn)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Quãng đường (nhỏ hơn)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Vận tốc (lớn hơn)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Vận tốc (nhỏ hơn)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gia tốc (lớn hơn)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gia tốc (nhỏ hơn)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Lực tác động (lớn hơn)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Lực tác động (nhỏ hơn)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Thời gian (lớn hơn)</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_Condition_Stop_Unit">
        <property name="geometry">
         <rect>
          <x>304</x>
          <y>56</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>(mm/s2)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="pushButton_Save">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>200</y>
         <width>181</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(225, 225, 225);</string>
       </property>
       <property name="text">
        <string>Lưu</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_New">
       <property name="geometry">
        <rect>
         <x>790</x>
         <y>200</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(225, 225, 225);</string>
       </property>
       <property name="text">
        <string>Tạo mới</string>
       </property>
      </widget>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>10</y>
         <width>381</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame{
border:1px solid gray;
background-color: rgb(44, 58, 70);
	color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QComboBox" name="comboBox_Condition_Start_Car">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>10</y>
          <width>91</width>
          <height>32</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox {
    border: 1px solid gray;
	color: rgb(255, 255, 255);
}
QComboBox::drop-down {
    width:35px;
}</string>
        </property>
        <item>
         <property name="text">
          <string>XE-01</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>XE-02</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>TẤT CẢ</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>14</y>
          <width>191</width>
          <height>22</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 170, 127);</string>
        </property>
        <property name="text">
         <string>Điều kiện bắt đầu cho</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_Condition_Start">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>50</y>
          <width>91</width>
          <height>32</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="maxLength">
         <number>4</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_Condition_Start_Unit">
        <property name="geometry">
         <rect>
          <x>304</x>
          <y>56</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>(mm/s2)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_Condition_Start">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>191</width>
          <height>32</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox {
    border: 1px solid gray;
	color: rgb(255, 255, 255);
}
QComboBox::drop-down {
    width:35px;
}</string>
        </property>
        <item>
         <property name="text">
          <string>Không có điều kiện</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Quãng đường (lớn hơn)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Quãng đường (nhỏ hơn)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Vận tốc (lớn hơn)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Vận tốc (nhỏ hơn)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gia tốc (lớn hơn)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Gia tốc (nhỏ hơn)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Lực tác động (lớn hơn)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Lực tác động (nhỏ hơn)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Thời gian (lớn hơn)</string>
         </property>
        </item>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>110</y>
         <width>911</width>
         <height>91</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox{
border:0px;
color: rgb(0, 255, 255);
}
</string>
       </property>
       <property name="title">
        <string>Các bài thực hành cơ bản</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>8</y>
          <width>661</width>
          <height>20</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>921</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QComboBox" name="comboBox_Unit">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>861</width>
           <height>32</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QComboBox {
    border: 1px solid gray;
	color: rgb(255, 255, 255);
}
QComboBox::drop-down {
    width:35px;
}</string>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QPushButton" name="pushButton_Delete">
       <property name="geometry">
        <rect>
         <x>630</x>
         <y>200</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(225, 225, 225);</string>
       </property>
       <property name="text">
        <string>Xóa</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_Run">
      <property name="geometry">
       <rect>
        <x>359</x>
        <y>420</y>
        <width>321</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	background-color: rgb(0, 85, 127);
    border-style: outset;
    border-width: 1px;
	border-color: rgb(0, 50, 74);
}
</string>
      </property>
      <property name="text">
       <string>Thực hiện</string>
      </property>
      <property name="shortcut">
       <string>Space</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_arrowBlink">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>50</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_Blink_Start">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 170, 127);</string>
      </property>
      <property name="text">
       <string>Điều kiện bắt đầu</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Run_Comment">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>1021</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <italic>true</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>Hướng dẫn</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>1001</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-repeat: no-repeat;
background-position: center;
background-image: url(:/image/MFS0103_Unit01.jpg);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Cancel">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>440</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(225, 225, 225);</string>
      </property>
      <property name="text">
       <string>Thoát</string>
      </property>
      <property name="shortcut">
       <string>Esc</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>          Cài đặt          </string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../image.qrc"/>
 </resources>
 <connections/>
</ui>
